data_structures/binary_tree/binary_search_tree.py

```diff
@@ -142,16 +142,26 @@ def traversal_tree(self, traversal_function=None):
             return traversal_function(self.root)
 
     def inorder(self, arr, Node):
+        """This function performs inorder traversal and append values of nodes to list named arr
+        Args:-
+        arr -> list
+        Node -> object of class Node
+        """
         if Node:
             self.inorder(arr, Node.left)
-            arr.append(Node.value)
+            arr.append(Node.value)  # appends values of nodes to list arr
```

Comment:
This comment merely repeats what the code already says.  It can be deleted.

Reply:
ok I will remove it.

Reply:
Removed repeated comments and wrote function name in small letters.